### PR TITLE
Swap use of some q* macros to std calls

### PR DESCRIPTION
**Pull request by nyalldawson**
Better to stick to standard methods where available instead of less supported Qt methods

### CODE REVIEW

File: src/analysis/raster/qgsalignraster.cpp [hunk 1]
```diff
@@ -30,16 +30,16 @@
 
 static double ceil_with_tolerance( double value )
 {
-  if ( qAbs( value - qRound( value ) ) < 1e-6 )
-    return qRound( value );
+  if ( fabs( value - std::round( value ) ) < 1e-6 )
```
Comment (jef-n) [inline src/analysis/raster/qgsalignraster.cpp]: Why qualify round but not fabs?

**Comment by luipir** [comment]
strongly agree!

**Comment by nyalldawson** [comment]
Travis failure is spell check timeout only. I can't get it to pass, regardless of how many times I restart the job.

**Comment by nyalldawson** [comment]
@jef-n this ok for you?

**Comment by jef-n** [comment]
LGTM - go ahead.  in case it breaks the windows build, I'll just fix what needs fixing.

**Comment by nyalldawson** [comment]
Thanks @jef-n!

**Pull request merged**